static/docker-compose.yml

```diff
@@ -48,24 +49,10 @@ services:
          npm run db:migrate &&
          npm run pg:etl up all &&
          npm run maintenance:disable"
+    env_file:
+    - .env
     environment:
-      COUCHBASE_URL: ${COUCHBASE_URL:-http://plextracdb}
-      CB_API_PASS: ${CB_API_PASS}
-      CB_API_USER: ${CB_API_USER}
-      REDIS_CONNECTION_STRING: ${REDIS_CONNECTION_STRING:-redis}
-      REDIS_PASSWORD: ${REDIS_PASSWORD:?err}
-      PG_HOST: ${PG_HOST:-postgres}
       PG_MIGRATE_PATH: /usr/src/plextrac-api
```

Comment:
It looks like the code defaults to this path for production builds, so this env is not useful outside of dev environments.